### PR TITLE
Add openshift/dpu-network-operator repo

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

**Comment by pliurh** [comment]
> I assume you just copied this from some other module? It all looks good to me but I don't know what's actually needed when adding a new module...

I followed https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/ and referred to other modules. I think it shall work as long as we can pass all the checks.

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 1]
```diff
@@ -102,6 +102,7 @@ registry.ci.openshift.org/origin/4.10:deployer quay.io/openshift/origin-deployer
 registry.ci.openshift.org/origin/4.10:descheduler quay.io/openshift/origin-descheduler:4.10 quay.io/openshift/origin-descheduler:4.10.0 quay.io/openshift/origin-descheduler:latest
 registry.ci.openshift.org/origin/4.10:docker-builder quay.io/openshift/origin-docker-builder:4.10 quay.io/openshift/origin-docker-builder:4.10.0 quay.io/openshift/origin-docker-builder:latest
 registry.ci.openshift.org/origin/4.10:docker-registry quay.io/openshift/origin-docker-registry:4.10 quay.io/openshift/origin-docker-registry:4.10.0 quay.io/openshift/origin-docker-registry:latest
+registry.ci.openshift.org/origin/4.10:dpu-ovnkube-operator quay.io/openshift/origin-dpu-ovnkube-operator:4.10 quay.io/openshift/origin-dpu-ovnkube-operator:4.10.0 quay.io/openshift/origin-dpu-ovnkube-operator:latest
```
[the remark this answers is not in the document]
Comment (smarterclayton) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:105]: If the operator is unrelated to ovnkube or will eventually cover something else, then `dpu-network-operator` is apporpriate when you're managing another cluster, or `cluster-dpu-network-operator` when the operator manages the current cluster.

[the remark this answers is not in the document]
Comment (pliurh) [reply core-services/image-mirroring/openshift/mapping_origin_4_10]: renamed to `dpu-network-operator`

**Comment by omertuc** [comment]
This PR deleted all of our `reporter_config` entries...

**Comment by zshi-redhat** [comment]
> This PR deleted all of our `reporter_config` entries...

It was deleted accidentally, thanks for adding them back, Omer!

**Comment by bparees** [comment]
> It was deleted accidentally, thanks for adding them back, Omer!

@omertuc  @zshi-redhat where did they get added back?

**Comment by bparees** [comment]
adding the canary job notifications back in here: https://github.com/openshift/release/pull/24387

**Pull request merged**